spec/features/material_compare_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "Comparing two materials" do
+  let(:material1) {
+    create(
+      :material,
+      name: "Cheese",
+      manufacturer: manufacturer,
+    )
+  }
+  let(:material2) {
+    create(
+      :material,
+      name: "Bacon",
+      manufacturer: manufacturer,
+    )
+  }
+  let(:manufacturer) { create(:manufacturer, name: "Big Pizza") }
```

Comment:
It doesn't look like the name is being used anywhere? If not I think you can drop the manufacturer from the create call and let the factory populate it.